core/controlplane/config/templates/cloud-config-worker

```diff
@@ -546,15 +557,19 @@ write_files:
       rkt run \
         --volume=ssl,kind=host,source=/etc/kubernetes/ssl,readOnly=false \
         --mount=volume=ssl,target=/etc/kubernetes/ssl \
+        {{- if .Experimental.ClusterTLSBootstrap.Enabled }}
+        --volume=kube,kind=host,source=/etc/kubernetes,readOnly=false \
+        --mount=volume=kube,target=/etc/kubernetes \
+        {{- end }}
```

Comment:
When the cluster TLS bootstrap feature is enabled, the original volume and the mount for `/etc/kubernetes/ssl` seems not be necessary; then, how about something like:

```
{{- if .Experimental.ClusterTLSBootstrap.Enabled }}
--volume=kube,kind=host,source=/etc/kubernetes,readOnly=false \
--mount=volume=kube,target=/etc/kubernetes \
{{- else}}
--volume=ssl,kind=host,source=/etc/kubernetes/ssl,readOnly=false \
--mount=volume=ssl,target=/etc/kubernetes/ssl \
{{- end }}
```

?